### PR TITLE
Update readme to include bowl and local viewing help

### PR DESCRIPTION
**Pull request by russellthorn**
To save others having to spend time asking how to bowl or what the
local url for smart answers is

### CODE REVIEW

**Comment by tadast** [comment]
:+1: :trophy:

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ NB: this assumes you are running on the GOV.UK virtual machine, not your host.
   ./install # git fetch from each dependency dir and bundle install
 ```
 
+Run using bowler:
+```
+bowl smartanswers
```
Comment (tadast) [inline README.md]: should we add `cd ../puppet/development` here?

Comment (russellthorn) [inline README.md]: You bowl from cd /var/govuk/development/ on the VM or at least I do.

Happy to update this to reflect that

Comment (russellthorn) [inline README.md]: I've updated that and added that to the original commit message.

**Comment by dsingleton** [comment]
:+1:

**Pull request merged**